openapi/components/schemas/RichBillingPortal.yaml

```
@@ -0,0 +1,140 @@
+type: object
+required:
+  - name
+  - recurringOptions
+  - levels
+  - solutions
+properties:
+  id:
+    type: string
+    description: ID of the billing portal.
+    readOnly: true
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
+  name:
+    type: string
+    description: Billing portal name.
```

nitpick: [noun string](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
    description: Name of the billing portal.
```